app/models/in_progress_form.rb

```diff
@@ -27,6 +29,8 @@ def cast(value)
 
   attribute :user_uuid, CleanUUID.new
   attr_encrypted :form_data, key: Settings.db_encryption_key
+  serialize :form_data, JsonMarshal::Marshaller
```

Comment:
InProgressForm record form_data is a Hash (verified in prod and staging). The form_data needs to be coerced into a string before saving into the db and retrieved as a Hash. The JsonMarshal::Marshaller class will properly save and retrieve in the expected form.